### PR TITLE
Add configs for inference

### PR DESCRIPTION
**Pull request by JoannaLXY**

### CODE REVIEW

File: configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    dict(
+        type='SampleFrames',
+        clip_len=32,
+        frame_interval=2,
+        num_clips=10,
+        test_mode=True),
+    dict(type='DecordDecode'),
+    dict(type='Resize', scale=(-1, 256)),
+    dict(type='ThreeCrop', crop_size=256),
+    dict(type='Flip', flip_ratio=0),
```
Comment (innerlee) [inline configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py:55]: remove if it is not used

File: configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    dict(type='DecordInit'),
+    dict(
+        type='SampleFrames',
+        clip_len=32,
+        frame_interval=2,
+        num_clips=10,
+        test_mode=True),
+    dict(type='DecordDecode'),
+    dict(type='Resize', scale=(-1, 256)),
+    dict(type='ThreeCrop', crop_size=256),
```
Comment (innerlee) [inline configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py:54]: double check 256 or 224

Comment (JoannaLXY) [reply configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py:54]: 256 is correct

Comment (innerlee) [reply configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py:54]: so, it train with 224 and test with 256?

Comment (kennymckormick) [reply configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py:54]: > so, it train with 224 and test with 256?

Yes, for 3D models this setting is good

Comment (innerlee) [reply configs/recognition/slowfast/slowfast_r50_video_inference_4x16x1_256e_kinetics400_rgb.py:54]: kind of poor man's ensemble

**Comment by innerlee** [comment]
look good

**Pull request merged**